custom/dhis2/payload_generators.py

```diff
@@ -68,6 +66,7 @@ def get_payload(self, repeat_record, form):
             logger.debug('DHIS2: Processing Register Child form')
             push_case(case, dhis2_api)
             # We just need to enroll. No event to create
+            raise IgnoreDocument
```

Comment:
Interestingly, lines 74 and 90 are situations where we want to try again later. They would happen when a register form hasn't been processed yet, but must exist. But I don't think lines 74 and 90 should raise errors as they do here, because they aren't totally unexpected. In [master][1] they now just `return`.

  [1]: https://github.com/dimagi/commcare-hq/blob/master/custom/dhis2/payload_generators.py#L76